### PR TITLE
fix(curriculum): build a city skyline instruction updated

### LINKED ISSUE #47555: feedback on Learn Css Variables By Building A City Skyline
### Describe the Issue

this project assumes you haven't worked with Flexbox yet hence it just givesyou the code without it being a real challenge/test.

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-css-variables-by-building-a-city-skyline

### Your code

```
none
```

### Expected behavior

just to be given directions about what to do (as per usual) without giving you all the lines of code.

### Screenshots

Screenshot at 2022-09-18 15-46-28

### System

- Device: Zenbook
- OS: Linux Mint 20.3
- Browser: Brave
- Version: v1.43.93

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by nayabatir1**
…kyline instruction updated

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Addresses #47555

### CODE REVIEW

**Review by Sembauke — CHANGES_REQUESTED** [review]
Hey @nayabatir1,

This does not look like our style of writing, can you look up a couple of challenges and 
see if you can give it the "freeCodeCamp" style?

Have a nice day and happy coding.

**Comment by nayabatir1** [comment]
Hey @Sembauke what do you mean by fcc style?
Can you share some examples for reference

**Comment by ieahleen** [comment]
Can you please use a descriptive title? the title of the issue is not a descriptive title for the PR

**Comment by nayabatir1** [comment]
@ieahleen can pls confirm, is it appropriate ?

File: curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98d5.md [hunk 1]
```diff
@@ -7,7 +7,9 @@ dashedName: step-13
 
 # --description--
 
-Give your `.bb1` element these style properties: `display: flex;`, `flex-direction: column;`, and `align-items: center;`. This will center the parts of the building using "flex" or "flexbox". You will learn about it in more detail on another project.
+1. Turn your `.bb1` element into a flexbox parent so you can center the parts of the building.<br>- hint: You should give `.bb1` a `display` of `flex`
```
Comment (ieahleen) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98d5.md:10]: The `hints` are what go in the next section to display when a test fail.

It seems you misinterpreted what was intended with "split in 3 different steps"

please check our documentation and look at how the project works in the curriculum.

**Comment by ieahleen** [comment]
And to add to that, it was decided to not divide in three steps, but to make the stepp less 'handolding', with the wording mentioned in https://github.com/freeCodeCamp/freeCodeCamp/issues/47555#issuecomment-1275090120

**Comment by nayabatir1** [comment]
@ieahleen I've updated the pr. Do let me know if I missed anything.

File: curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98d5.md [hunk 2]
```diff
@@ -19,21 +19,21 @@ assert.equal(bb1Style?.width, '10%');
 assert.equal(bb1Style?.height, '70%');
 ```
 
-You should give the `.bb1` element a `display` of `flex`.
+You should give `.bb1` a `display` of `flex`.
```
Comment (ieahleen) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98d5.md:22]:
```suggestion
You should give the `.bb1` element a `display` of `flex`.
```

File: curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98d5.md [hunk 3]
```diff
@@ -19,21 +19,21 @@ assert.equal(bb1Style?.width, '10%');
...
 
 ```js
 const bb1Style = new __helpers.CSSHelp(document).getStyle('.bb1');
 assert.equal(bb1Style?.display, 'flex');
 ```
 
-You should give the `.bb1` element a `flex-direction` of `column`.
+`.bb1` should have a `flex-direction` property.<br>You should give `.bb1` a `flex-direction` of `column`
```
Comment (ieahleen) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98d5.md:29]:
```suggestion
You should give the `.bb1` element a `flex-direction` of `column`.
```

File: curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98d5.md [hunk 4]
```diff
@@ -19,21 +19,21 @@ assert.equal(bb1Style?.width, '10%');
...
 
 ```js
 const bb1Style = new __helpers.CSSHelp(document).getStyle('.bb1');
 assert.equal(bb1Style?.flexDirection, 'column');
 ```
 
-You should give the `.bb1` element a `align-items` of `center`.
+Your `.bb1` should have an `align-items` property.<br>You should give `.bb1` an `align-items` property of `center`
```
Comment (ieahleen) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98d5.md:36]:
```suggestion
You should give the `.bb1` element a `align-items` of `center`.
```

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
Please follow the decision in the issue, do not do random changes

**Comment by nayabatir1** [comment]
> Please follow the decision in the issue, do not do random changes

tbh I copied the changes from here
image

**Comment by ieahleen** [comment]
That was the breakdown for splitting the challenge in three different steps, as it's not what it was decided to do, you don't need to take that wording

**Review by naomi-lgbt — APPROVED** [review]
banger pr, thanks for contributing!

**Review by raisedadead — APPROVED** [review]

**Pull request merged**